docs/tutorial/imagestack.rst

```diff
@@ -5,113 +5,143 @@ Image Stacks
 
-    stack = lk.ImageStack("wf.tiff")  # Loading a stack.
-    stack_slice = stack[2:10]  # Grab frame 2 to 9
+    stack = lk.ImageStack("stack.tiff")  # Loading a stack.
 
-You can also slice by time::
+Long videos are exported as a set of multiple files due to the size constraint on the TIF format.
+In this case, you can supply the filenames consecutively to treat them as a single object::
 
-    stack_slice = stack["2s":"10s"]  # Slice from 2 to 10 seconds from beginning of the stack
+    # long_stack = lk.ImageStack("file1.tiff", "file2.tiff")
 
-You can easily load multiple TIFF files by simply listing them consecutively::
+Full color RGB fluorescence images recorded with the widefield or TIRF functionality on C-Trap Edge systems
+are automatically reconstructed using the alignment matrices from Bluelake, if available. This functionality can be
+turned off with the optional `align` keyword. Note that the `align` parameter has to be provided as a keyworded argument::
 
-    stack = lk.ImageStack("wf.tiff", "wf2.tiff")  # Loading two tiff files in a single stack.
+    raw_stack = lk.ImageStack("stack.tiff", align=False)
 
-You can quickly plot an individual frame using the
-:meth:`~lumicks.pylake.ImageStack.plot()` method::
 
-    stack.plot(frame=0, channel="rgb")
+Slicing and cropping
+--------------------
 
-.. image:: correlatedstack_aligned.png
+We can slice the stack by frame index::
 
-Sometimes a few bright pixels can dominate the image.
-When this is the case, it may be beneficial to manually set the color limits for each of the channels.
-This can be accomplished by providing a :class:`~lumicks.pylake.ColorAdjustment` to plotting or export functions::
+    stack_slice = stack[2:10]  # Grab frame 3 to 9
 
-    stack.plot(channel="red", adjustment=lk.ColorAdjustment([50, 50, 50], [100, 250, 196]))
+or by time::
 
+    stack["2s":"10s"]  # Slice from 2 to 10 seconds from beginning of the stack
 
-There are also a number of custom colormaps for plotting single channel images. These are available from :data:`~lumicks.pylake.colormaps`; the available colormaps are:
-`.red`, `.green`, `.blue`, `.magenta`, `.yellow`, and `.cyan`. For example, we can plot the blue channel image with the cyan colormap::
-
-    stack.plot(channel="blue", cmap=lk.colormaps.cyan)
+You can also spatially crop to select a smaller region of interest::
 
-By default the limits should be provided in absolute values, although percentiles can be used instead for convenience::
+    stack_roi = stack_slice.crop_by_pixels(10, 400, 10, 120)  # pixel coordinates as x_min, x_max, y_min, y_max
 
-    stack.plot(channel="red", adjustment=lk.ColorAdjustment([5, 5, 5], [95, 95, 95], mode="percentile"))
+Cropping can be useful, for instance, after applying color alignment to RGB images as the edges
+become corrupted due to interpolation artifacts. We can see this by plotting just the `"red"` channel
+(details of the plotting method are explained in the next section)::
```

Comment:
Hm, when you say interpolation, I'd really think of artefacts in the part of the image that has pixel data (but that isn't what you mean here I think).

```suggestion
Cropping can be useful, for instance, after applying color alignment to RGB images since many
of the pixels near the edges don't have data. We can see this by plotting just the `"red"` channel
(details of the plotting method are explained in the next section)::
```